stellarphot/settings/models.py

```diff
@@ -248,17 +250,20 @@ class PhotometryApertures(BaseModel):
     ... )
     """
 
-    radius: conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
-    gap: conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
-    annulus_width: conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
+    # model_config = MODEL_DEFAULT_CONFIGURATION
+
```

Comment:
I made two changes here that you'll see when I push the code in a bit:

1. Add notes to the docstrings indicating values must be positive 
2. Used `PositiveInt` and `PositiveFloat` from pydantic 2 instead of `conint(ge=1)` and `confloat(ge=0)`. The former are much less opaque than the latter